composer.json

```diff
@@ -37,7 +37,10 @@
         "phpunit/dbunit": "~1.4",
         "symfony/phpunit-bridge": "~2.8|~3.0|^4.0",
         "squizlabs/php_codesniffer": "^2.7",
-        "m6web/Symfony2-coding-standard": "^3.3"
+        "m6web/Symfony2-coding-standard": "^3.3",
+        "florianv/exchanger": "^1.0|^2.0",
```

Comment:
Can you let the composer.json unchanged for this PR. I believe this modification should be in another PR.

I'll check the other PR.

Reply:
hum, nope, forget my comment... I didn't read your PR close enough...

It's ok for the fix I guess